src/main/java/io/github/moulberry/notenoughupdates/profileviewer/GuiProfileViewer.java

```diff
@@ -539,16 +574,12 @@ public void printStackTrace() {
 	}
 
 	private void renderTabs(boolean renderPressed) {
-		int ignoredTabs = 0;
-		for (int i = 0; i < ProfileViewerPage.values().length; i++) {
-			ProfileViewerPage page = ProfileViewerPage.values()[i];
-			if (page.stack == null) {
-				ignoredTabs++;
-				continue;
-			}
+		final List<ProfileViewerPage> pages = ProfileViewerPage.getPagesWithItem();
+		for (int index = 0; pages.size()>index; index++){
```

Comment:
```suggestion
		for (int index = 0; pages.size() > index; index++){
```
:slightly_smiling_face: